components/brave_wallet_ui/page/screens/swap/hooks/useSwap.ts

```diff
@@ -501,9 +545,15 @@ export const useSwap = () => {
   //     debouncing.
   //  5. Fetch spot price.
   const onSelectToToken = useCallback(
+    // ToDo: Add account as a prop once we support cross-chain swaps.
```

Comment:
How does an `account` prop help here?

Reply:
Was assuming that the `onSelectToToken` method would also set the `toAccount` when selecting a token in the new `Composer UI`. Removed this comment!